yt/testing.py

```diff
@@ -431,7 +431,14 @@ def fake_random_ds(
 
 
 def fake_amr_ds(
-    fields=None, units=None, geometry="cartesian", particles=0, length_unit=None
+    fields=None,
+    units=None,
+    geometry="cartesian",
+    particles=0,
+    length_unit=None,
+    *,
+    domain_left_edge=None,
+    domain_right_edge=None,
```

Comment:
I needed to expose these parameters because we didn't have a fake_ds function that exposed both domain limits and geometry, which explains why the cases I'm interested in here are not covered yet.